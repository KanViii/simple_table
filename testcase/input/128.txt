ASSIGN a 4n123