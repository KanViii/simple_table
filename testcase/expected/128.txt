Invalid: ASSIGN a 4n123